pkg/workload/schemachange/operation_generator.go

```diff
@@ -3916,6 +3923,9 @@ func (og *operationGenerator) createFunction(ctx context.Context, tx pgx.Tx) (*o
 			name := tree.Name(fmt.Sprintf("udf_%s", og.newUniqueSeqNumSuffix()))
 			return &name
 		},
+		"Schema": func() (string, error) {
```

Comment:
oh just thought of something: is it worth to check if our schema exists? i'm asking in the case where we have selected our descriptors to get `descJSONQuery`, another worker fires of a drop schema, and then we try to create a function with that schema here

Reply:
i think we're safe from this, we use same transaction to pick the schema name and create the function in that schema, so it's guaranteed that no other transaction will have dropped that schema, thanks to serializable transaction guarantees

Reply:
oh true - ty for explaining why 🥇